lib/payoneer/client.rb

```diff
@@ -39,14 +39,62 @@ def payout(program_id:, payment_id:, payee_id:, amount:, description:, payment_d
       )
     end
 
+    # Includes additional items as needed to be Payoneer SAFE compliant (https://app.asana.com/0/440902128870182/473284740903863/f)
+    def expanded_payout(payee_id:, client_reference_id:, amount:, currency: 'USD', description:, payout_date: Time.now, seller_id:, seller_name:, seller_url:, seller_type: 'ECOMMERCE', path:)
+      params = {
+        payee_id: payee_id,
+        client_reference_id: client_reference_id,
+        amount: amount,
+        currency: currency,
+        description: description,
+        payout_date: payout_date.strftime('%Y-%m-%d'),
+        orders_report: {
+          merchant: {
+            id: seller_id,
+            store: {
+              name: seller_name,
+              url: seller_url,
+              type: seller_type
+            }
+          },
+          orders: {
+            type: 'url',
+            path: path,
+            credentials: { # @TODO: What type of credentials? When should we expire?
+              type: 'AUTHORIZATION',
+              token: 'FILL_ME_IN'
```

Comment:
This is TBD. We will wait on Payoneer to figure out what to do about the credentials.